### DISCUSSION TITLE
Speed overhang management on version 1.6.4 Beta

### DISCUSSION THREAD

**Discussion by Zuffdesign** [OrcaSlicer/OrcaSlicer#1699] [General]
Hello,

First of all, thank you for this new Beta version!

I have a small problem with the speed management on overhangs.
In version 1.6.3 the speed difference was progressive.
https://ibb.co/4WGPjJM

Now with version 1.6.4 Beta the speed changes abruptly, which can be seen on the printouts.
https://ibb.co/w0QH1sg
https://ibb.co/f263DR3

Is there a parameter to change for this? Could it be corrected to be progressive again?

Thank you for your reply.
Sorry for my English, I'm using a translator.

Best regards

Zuff

**Comment by SoftFever** [comment]
Hi @Zuffdesign 

1.6.4 revert back to the classic overhang slowdown approach which dont' the progressive speed changing.
In my printing test, I found it makes no difference.

**Reply by Zuffdesign** [reply]
Thank you for the answer but I'm not sure I understood... You are going to modify version 1.6.4 with the same progressive operation of overhangs as on the old version 1.6.3?

**Reply by SoftFever** [reply]
No, I meant 1.6.4 onwards won't have that progressive speed.

**Reply by Zuffdesign** [reply]
Ah ok, it's a shame because the quality of the prints is less good as on the link of my photo where you can clearly see the demarcation that corresponds to the demarcation that has been distinguished on the slicer...
I think it will show even more on silk filaments.
